plugin/inc/views/printers.class.php

```diff
@@ -194,7 +194,7 @@ public static function getTicketPrinterDisplay($row_data, $import_data): string
             $style   .= 'font-style: italic;';
         }
 
-        if ($row_data['qr_code']) {
+        if (isset($row_data['qr_code'])) {
```

Comment:
Do we have to display [QR] if the qr_code field is set but empty?
```suggestion
        if ($row_data['qr_code'] ?? false) {
```